core/inject.js

```diff
@@ -55,9 +56,15 @@ Blockly.inject = function(container, opt_options) {
   var subContainer = goog.dom.createDom('div', 'injectionDiv');
   container.appendChild(subContainer);
   var svg = Blockly.createDom_(subContainer, options);
+
+  // Create surfaces for dragging things. These are optimizations
+  // so that the broowser does not repaint during the drag.
   var blockDragSurface = new Blockly.BlockDragSurfaceSvg(subContainer);
   blockDragSurface.createDom();
```

Comment:
For followup: move createDom into BlockDragSurfaceSvg constructor, as you're doing for the workspace drag surface.